packages/cli/src/commands/prisma.ts

```diff
@@ -100,16 +97,11 @@ export const builder = async (yargs: Argv) => {
     // So we can check for yarn prisma in the output
     const { stdout } = await prismaCommand
 
-    // Show prisma cli output
-    // console.log(stdout)
-
-    if (hasHelpFlag || stdout.match('yarn prisma')) {
+    if (hasHelpFlag || stdout?.match('yarn prisma')) {
```

Comment:
I don't really understand the second condition here? We're not running prisma via yarn, so is it checking for something else?

Reply:
It's checking the output from the prisma cli, because the cli sometimes says stuff like "generated xxx, now run yarn prisma to yyy"

Reply:
I actually see that the Prisma CLI always outputs something like this right at the top:
```
$ yarn rw prisma generate

Running prisma cli:
yarn prisma generate --schema "/private/var/folders/jj/brzmsc2562zbwgzwbnjdnqp80000gn/T/redwood.dQPMMURv/api/db/schema.prisma"
```

So, shouldn't this be showing the box on every command? 🤔

Reply:
So that actually comes from our cli at this line https://github.com/redwoodjs/redwood/blob/2a1b4c6495a4933e5e2c5fd46d2e4e292011c57f/packages/cli/src/commands/prisma.ts#L77. That's why I do a check of the stdout from the prisma cli's output specifically

Reply:
lol, wtf! I swear searched for this in our code before writing this comment... Could we change that up to read "Running Prisma CLI"

Reply:
Hmpf. I'm trying to find the commands that ask you to do this, is it when you save a migration?

Reply:
Anyway, don't want to bike-shed here too much. I approve this PR, but I would love it if you could document which commands would cause this trigger.

It might be that in future releases of Prisma they may not trigger this anymore, and we'll have this code lying around for no reason. It may also be easier to check the sequence of commands rather than the output.

Reply:
Here's one `prisma introspect` -> https://s.tape.sh/7UcUY1t8 But I've seen it elsewhere (maybe migrate --create-only?)

Reply:
The original PR actually has more in the description btw https://github.com/redwoodjs/redwood/pull/1772

Reply:
🆒 🆒 🆒 , I think documenting this is super important for future readers of the code

Reply:
Since they don't have to wonder about what triggers it if they wanted to test it. They could just read a comment and figure, ah ok, I can test this by running `yarn rw prisma introspect`